src/javascript/app/common/deriv_banner.js

```diff
@@ -0,0 +1,50 @@
+const DerivBanner = (() => {
+    let $deriv_banner_countainer,
```

Comment:
we only use the dollar sign for jquery elements, please use `el_` prefix for these instead